test/functional/test_runner.py

```diff
@@ -145,6 +145,7 @@
     'feature_blocksdir.py',
     'feature_config_args.py',
     'feature_help.py',
+    'rpc_psbt.py',
```

Comment:
```
rpc_psbt.py                           | ✓ Passed  | 4 s
.
.
.
rpc_users.py                          | ✓ Passed  | 3 s
```

Since `rpc_psbt.py` ran locally for me in 4 seconds and to avoid possible merge conflict, `'rpc_psbt.py'` can be inserted before `'rpc_users.py'` in this list.

Reply:
Done